apps/frontend/app/routes/_dashboard.media.item.$id._index.tsx

```diff
@@ -1352,6 +1352,11 @@ const HistoryItem = (props: { history: History; index: number }) => {
 						e.episodeNumber === props.history.showExtraInformation?.episode,
 				)
 		: null;
+	const isNumberOrDecimalString = (value: unknown): boolean =>
+		typeof value === "number" ||
+		(typeof value === "string" &&
+			!Number.isNaN(Number.parseFloat(value)) &&
```

Comment:
re-export `isNan` and `isFinite` from lodash from `@ryot/ts-utils` and use here.